NMS-14311: Device Config Backup doc updates for config type (#4961)
OpenNMS/opennms: docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc

If you want to enable device config backup on existing nodes in a requisition, select the desired requisition and edit the node definition.
. Click on the edit icon and click *Add Node*.
. Set up your requisition as described in xref:operation:provisioning/getting-started.adoc#requisition-create[Create a requisition].

> . Add `DeviceConfig` service on the interface where we are backing up the config. If device has multiple config types, you can add services with prefix `DeviceConfig-`. For example to specify a "running" configuration (i.e. configuration that the device currently has while running), create a service with the name `DeviceConfig-running`. This will display in the UI as "Running Configuration". If you specify `DeviceConfig-running-xml`, this will display in the UI as "Running Xml Configuration".

Reviewer: The + sign will allow for a line break to still be part of the same bullet/number item.

> . Add `DeviceConfig` service on the interface where we are backing up the config. If device has multiple config types, you can add services with prefix `DeviceConfig-`. For example to specify a "running" configuration (i.e. configuration that the device currently has while running), create a service with the name `DeviceConfig-running`. This will display in the UI as "Running Configuration". If you specify `DeviceConfig-running-xml`, this will display in the UI as "Running Xml Configuration".

Reviewer 2: Thanks for clarifying the syntax, Mark.

> . Add `DeviceConfig` service on the interface where we are backing up the config. If device has multiple config types, you can add services with prefix `DeviceConfig-`. For example to specify a "running" configuration (i.e. configuration that the device currently has while running), create a service with the name `DeviceConfig-running`. This will display in the UI as "Running Configuration". If you specify `DeviceConfig-running-xml`, this will display in the UI as "Running Xml Configuration".

Reviewer: ```suggestion
. Add `DeviceConfig` service on the interface where we are backing up the config.
.. If a device has multiple config types, use the prefix `DeviceConfig-` to add services. +
For example, use `DeviceConfig-running` to specify a configuration that the device currently has while running.
The UI displays this as "Running Configuration". +
For `DeviceConfig-running-xml`, the UI displays "Running Xml Configuration".
```

> . Add `DeviceConfig` service on the interface where we are backing up the config. If device has multiple config types, you can add services with prefix `DeviceConfig-`. For example to specify a "running" configuration (i.e. configuration that the device currently has while running), create a service with the name `DeviceConfig-running`. This will display in the UI as "Running Configuration". If you specify `DeviceConfig-running-xml`, this will display in the UI as "Running Xml Configuration".

Reviewer: The above renders as:
![image](https://user-images.githubusercontent.com/26419758/176514648-95f210c1-f292-4317-9985-7f0092349016.png)

> . Add `DeviceConfig` service on the interface where we are backing up the config. If device has multiple config types, you can add services with prefix `DeviceConfig-`. For example to specify a "running" configuration (i.e. configuration that the device currently has while running), create a service with the name `DeviceConfig-running`. This will display in the UI as "Running Configuration". If you specify `DeviceConfig-running-xml`, this will display in the UI as "Running Xml Configuration".

Reviewer 2: That's perfect.

. Click *Add Meta-Data*, and define the following key-value pairs:

> .. `dcb:username`: username for the device. This can also be an SCV metadata expression.

Reviewer 2: Make "SCV metadata expressions" a link to the SCV metadata expressions documentation.
Will need to update the /operation/pages/meta-data.adoc file to add an anchor tag to the SCV topic, then link to it. (I think this is the right link, unless there is other SCV documentation that is more applicable.)

> .. `dcb:password`: password for the device. This can also be an SCV metadata expression.

Reviewer 2: no need to add link here, as the one in line 23 is right above.

Author: Done
